### DISCUSSION TITLE
Formal rule on insider builds and preview updates vs releases

### DISCUSSION THREAD

**Discussion by pyrates999** [valinet/ExplorerPatcher#1785] [Q&A] [answered]
I'm wondering.  Do the developers of explorer patcher do any testing of explorer patcher on insider builds and releases with preview updates installed?

Or do they only test against releases with normal updates installed?

**Comment by vippium** [comment]
Let @valinet to have a eye on this ❓❓

**Comment by pyrates999** [comment, marked as answer]
From what @Poopooracoocoo has been answering, preview updates and insider builds are not supported by explorer patcher.

**Reply by Poopooracoocoo** [reply]
Yeah it's in https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems which is linked to from https://github.com/valinet/ExplorerPatcher/discussions/622

**Comment by pyrates999** [comment]
Sources:

https://github.com/valinet/ExplorerPatcher/issues/1896#issuecomment-1689006850
https://github.com/valinet/ExplorerPatcher/issues/1890#issuecomment-1687868704
